src/containers/home/index.tsx

```diff
@@ -118,38 +125,25 @@ const Home: React.FC<HomeContainerProps> = ({ announcements }) => {
           <ViewMoreButton to="upcoming-events">View All Events</ViewMoreButton>
         </Row>
         <Row gutter={[24, 24]} justify="center">
-          <Col>
-            <EventCard
-              src="https://www.pets4homes.co.uk/images/classifieds/2013/07/16/362464/large/beautiful-two-cats-bengal-x-british-shorthair-51f9b081db1b2.jpg"
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-              for a peaceful and rejuvenating virtual. Find some time for self-care and join yoga teacher Sarah Oleson
-              for a peaceful and rejuvenating virtual. Find some time for self-care and join yoga teacher Sarah Oleson
-              for a peaceful and rejuvenating virtual.`}
-              to="/"
-            />
-          </Col>
-          <Col>
-            <EventCard
-              src="http://thewowstyle.com/wp-content/uploads/2015/04/6891272-cats.jpg"
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-              for a peaceful and rejuvenating virtual...`}
-              to="/"
-            />
-          </Col>
-          <Col>
-            <EventCard
-              src="https://americanhumane.org/app/uploads/2016/08/animals-cats-cute-45170-min.jpg"
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-              for a peaceful and rejuvenating virtual...`}
-              to="/"
-            />
-          </Col>
+          {asyncRequestIsFailed(events) && (
+            <p>The announcements could not be retrieved.</p>
+          )}
+          {asyncRequestIsLoading(announcements) && <p>Loading events...</p>}
```

Comment:
`asyncRequestIsLoading(events)`*

Reply:
Oo yeah this was sloppy, this should be fixed